azurerm/resource_arm_kubernetes_cluster.go

```diff
@@ -1335,6 +1445,39 @@ func flattenKubernetesClusterNetworkProfile(profile *containerservice.NetworkPro
 		values["load_balancer_sku"] = string(profile.LoadBalancerSku)
 	}
 
+	lbProfiles := make([]interface{}, 0)
+
+	if profile.LoadBalancerProfile != nil {
+		lb := make(map[string]interface{})
+
+		if profile.LoadBalancerProfile.ManagedOutboundIPs != nil {
+			lb["managed_outbound_ips"] = profile.LoadBalancerProfile.ManagedOutboundIPs.Count
```

Comment:
since `.Count` here is a pointer we need to de-reference this ourselves unfortunately; as such could we make this:

```suggestion
			if profile.LoadBalancerProfile.ManagedOutboundIPs.Count != nil {
				lb["managed_outbound_ips"] = profile.LoadBalancerProfile.ManagedOutboundIPs.Count
			}
```